source/_integrations/androidtv_remote.markdown

```diff
@@ -75,99 +66,34 @@ Example:
 # Change channel to number 15:
 service: media_player.play_media
 data:
-  media_content_type: channel
+  media_content_type: "channel"
   media_content_id: 15
 target:
   entity_id: media_player.living_room_tv
 ```
 
-## Remote
+## Entity
 
-The remote allows you to send key commands to your Android TV device with the `remote.send_command` service.
+This integration adds a `remote` entity which turns on/off the Android TV device.
 The entity has the `current_activity` attribute that shows the current foreground app on the Android TV.
 
-{% details "List of the most common commands" %}
-
-Navigation:
-- DPAD_UP
-- DPAD_DOWN
-- DPAD_LEFT
-- DPAD_RIGHT
-- DPAD_CENTER
-- BUTTON_A
-- BUTTON_B
-- BUTTON_X
-- BUTTON_Y
-
-Volume Control:
-- VOLUME_DOWN
-- VOLUME_UP
-- VOLUME_MUTE
-- MUTE
-
-Media Control:
-- MEDIA_PLAY_PAUSE
-- MEDIA_PLAY
-- MEDIA_PAUSE
-- MEDIA_NEXT
-- MEDIA_PREVIOUS
-- MEDIA_STOP
-- MEDIA_RECORD
-- MEDIA_REWIND
-- MEDIA_FAST_FORWARD
-
-TV Control:
-- 0
-- 1
-- 2
-- 3
-- 4
-- 5
-- 6
-- 7
-- 8
-- 9
-- DEL
-- ENTER
-- CHANNEL_UP
-- CHANNEL_DOWN
-- F1
-- F2
-- F3
-- F4
-- F5
-- F6
-- F7
-- F8
-- F9
-- F10
-- F11
-- F12
-- TV
-- PROG_RED
-- PROG_GREEN
-- PROG_YELLOW
-- PROG_BLUE
-
-Other:
-- BUTTON_MODE
-- EXPLORER
-- MENU
-- INFO
-- GUIDE
-- TV_TELETEXT
-- CAPTIONS
-- DVR
-- MEDIA_AUDIO_TRACK
-- SETTINGS
-- SEARCH
-- ASSIST
+## Services
```

Comment:
Change this to ###? I think this would be better to be under the Remote section.

Reply:
Same as https://github.com/home-assistant/home-assistant.io/pull/27308#discussion_r1188366143
I just reverted the changes before https://github.com/home-assistant/home-assistant.io/pull/27059.